.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,34 @@
+name: Request a Feature
+description: Use this template to request a new feature
+title: "[Feature Request]: "

```

Comment:
here as well.

Reply:
Yes sure (thought the prefix would make it easier to distinguish and also adhere to the original issue template which has the [Feature Request] prefix)

Reply:
thanks for trying to be consistent! i actually never liked that about the existing feature request template. titles are for content, not for classifying issues. that's what issue labels are for. :)